ckan/templates/package/base.html

```diff
@@ -8,9 +8,9 @@
 
 {% block breadcrumb_content %}
   {% if pkg %}
-    {% set dataset = pkg.title or pkg.name %}
+    {% set dataset = h.dataset_display_name(pkg) %}
     {% if pkg.organization %}
-      {% set organization = pkg.organization.title or pkg.organization.name %}
+      {% set organization = h.get_translated(pkg.organization, 'title') or h.get_translated(pkg.organization, 'name') %}
```

Comment:
hmm. use h.dataset_display_name again? and maybe rename it to h.translated_display_name?